plugins/woocommerce/client/admin/client/settings-payments/components/offline-gateway-list-item/offline-payment-gateway-list-item.tsx

```diff
@@ -52,7 +52,9 @@ export const OfflinePaymentGatewayListItem = ( {
 							id={ gateway.id }
 							isOffline={ true }
 							enabled={ gateway.state.enabled }
-							settingsUrl={ gateway.management.settings_url }
+							settingsUrl={
+								gateway.management._links.settings.href
```

Comment:
I noticed that in all other instances, management is optional (?). Should we make it optional here as well?

Reply:
The thing is offline PMs will always have it (as defined in the type).